### PR TITLE
Adding POLY2 table for GF180 DRC

### PR DESCRIPTION
**Pull request by FaragElsayed2**
## Adding POLY2 table for GF180 DRC

### CODE REVIEW

**Comment by FaragElsayed2** [comment]
@atorkmabrains Ready to review

File: klayout/drc/rule_decks/poly2.drc [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  # Rule PL.9: Poly2 inter connect connecting 3.3V and 5V areas (area inside and outside Dualgate)
+  ## are not allowed. They shall be done though metal lines only.
+  logger.info('Executing rule PL.9')
+  pl9_l1 = poly2.interacting(poly2.not(v5_xtor).not(dualgate)).interacting(poly2.and(dualgate))
+  pl9_l1.output('PL.9',
+                'PL.9 : Poly2 inter connect connecting 3.3V and 5V areas (area inside and outside Dualgate)
+                 are not allowed. They shall be done though metal lines only.')
+  pl9_l1.forget
+
+  # rule PL.10 is not a DRC check
```
Comment (atorkmabrains) [inline klayout/drc/rule_decks/poly2.drc:164]: @FaragElsayed2 Please add link.

**Review by atorkmabrains — CHANGES_REQUESTED** [review]
@FaragElsayed2 Please check

File: klayout/drc/rule_decks/poly2.drc (hunk 1 above)
Comment (FaragElsayed2) [reply klayout/drc/rule_decks/poly2.drc]: Done @atorkmabrains

**Review by atorkmabrains — APPROVED** [review]

**Pull request merged**